validation/glean/baseline.1.core.pass.json

```diff
@@ -8,7 +8,8 @@
     "client_id": "6ff20eb7-e80d-4452-b45f-2ea7e63547aa",
     "seq": 1,
     "start_time": "2018-10-23 11:23:15-04:00",
-    "end_time": "2018-10-23 11:23:15-04:25"
+    "end_time": "2018-10-23 11:23:15-04:25",
```

Comment:
nit: Can we rename this file `baseline.1.pass.json`?

Reply:
Sure.

Reply:
Note: from [the readme](https://github.com/mozilla-services/mozilla-pipeline-schemas/blame/dev/README.md#L61) validation files should be of the form `<ping type>.<version>.<test name>.pass.json`, where `test name` is some short description to differentiate between different test cases.